frame/oracle/src/lib.rs

```diff
@@ -204,6 +210,13 @@ pub mod pallet {
 	pub type AssetsCount<T: Config> =
 		StorageValue<_, u32, ValueQuery, Nonce<ZeroInit, SafeIncrement>>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn reward_rate)]
+	#[allow(clippy::disallowed_types)]
+	/// Annualized reward rate for oracles. This is used to inflate currency supply when issuing the
+	/// reward per each block.
+	pub type RewardRate<T: Config> = StorageValue<_, Perbill, ValueQuery>;
```

Comment:
likely this is not enough as kt does not tracks previos issue (bias and slope in parlance of curve afaik)

Reply:
so we move reward rates into each pallet?